src/services/preference.service.js

```diff
@@ -0,0 +1,44 @@
+import models from '../models';
+
+const { Preferences } = models;
+
+/**
+ * @class PreferenceService
+ */
+export default class PreferenceService {
+  /**
+   * @static
+   * @param {*} preference
+   * @memberof NotificationService
+   * @return {notification} new notification
+   */
+  static async createPreference(preference) {
+    return Preferences.create(preference);
+  }
+
+  /**
+   * @static
+   * @param {*} property
+   * @memberof PreferenceService
+   * @return {notification} preferences
+   */
+  static findPreferences(property) {
```

Comment:
There is a mismatch here. The function name is in the plural form but it's only going to return one preference for a particular user.